src/Microsoft.ML.Transforms/Text/doc.xml

```diff
@@ -0,0 +1,165 @@
+          <para>Currently it supports only English.</para>
+      </remarks>
+      <example>
+        <code>
+          pipeline.Add(new SentimentAnalyzer(){ Source = &quot;TextColumn&quot; }  );
+        </code>
+      </example>
+    </member>
+
+    <member name="CharacterTokenizer">
+      <summary>
+        This transform breaks text into individual tokens, each consisting of individual character.
+      </summary>
+      <remarks>
+      This transform is not typically used on its own, but it is one of the transforms composing the 
+      <see cref="Microsoft.ML.Transforms.TextFeaturizer">Text Featurizer</see>. 
+      </remarks>
+      <seealso cref="Microsoft.ML.Transforms.WordTokenizer"/>
+      <seealso cref="Microsoft.ML.Transforms.TextToKey"/>
+      <seealso cref="Microsoft.ML.Transforms.NGramTranslator"/>
+      <seealso cref="Microsoft.ML.Transforms.TextFeaturizer"/>
+      <example>
+        <code>
+          pipeline.Add(new CharacterTokenizer("TextCol1" , "TextCol2" ) );
+        </code>
+      </example>
+    </member>
+
+    <member name="LightLDA">
+      <summary>
+        The LDA transform implements LightLDA, a state-of-the-art implementation of Latent Dirichlet Allocation.
+      </summary>
+      <remarks>
+        Latent Dirichlet Allocation is a well-known topic modeling algorithm that infers topical structure from text data,
+        and can be used to featurize any text fields as low-dimensional topical vectors. 
+        <para>LightLDA is an extremely efficient implementation of LDA developed in MSR-Asia that incorporates a number of 
+         optimization techniques. See <a href="http://arxiv.org/abs/1412.1576">LightLDA: Big Topic Models on Modest Compute Clusters</a>.
+        </para>
+        <para>
+          With the LDA transform, ML.NET users can train a topic model to produce 1 million topics with 1 million vocabulary
+          on a 1-billion-token document set one a single machine in a few hours (typically, LDA at this scale takes days and requires large clusters).
+          The most significant innovation is a super-efficient O(1) <a href="https://en.wikipedia.org/wiki/Metropolis–Hastings_algorithm">Metropolis-Hastings sampling algorithm</a>,
+          whose running cost is (surprisingly) agnostic of model size,
+          allowing it to converges nearly an order of magnitude faster than other <a href="https://en.wikipedia.org/wiki/Gibbs_sampling">Gibbs samplers.</a>
+        </para>
+        <para>
+          For more details please see original LightLDA paper, and its open source implementation. 
+          <list>
+            <item><description><a href="http://arxiv.org/abs/1412.1576"> LightLDA: Big Topic Models on Modest Computer Clusters</a></description></item>
+            <item><description><a href=" https://github.com/Microsoft/LightLDA">LightLDA </a></description></item>
+          </list>
+        </para>
+      </remarks>
+      <example>
+        <code>
```

Comment:
:bulb: It wouldn't hurt to include the `language` attribute for code blocks. Some tools don't use it (or infer a default that matches the implementation), but others use it for things like syntax highlighting. At worst the value is ignored.

```xml
<code language="csharp">
```